docs/content/dagster-cloud/managing-deployments/setting-up-alerts.mdx

```diff
@@ -96,6 +85,23 @@ Currently, Slack and email notifications are supported.
        height={514}
        />
 
+   For asset-based alerts, fill out these additional options:
+
+   - **Asset group** - Select the asset group to monitor. You will have the option to select from all asset groups in the deployment.
+   - **Asset key** - Select the asset key to monitor. You will have the option to select from all asset keys in the deployment. **Note:** If you select an asset group, you will not be able to select an asset key.
+   - **Events** - Select whether the alert should trigger on asset materialization failure, asset materialization success, asset check error, asset check warn, asset check passed, or asset check failure to execute
+   - **Notification Service** - Asset alerts have an additional notification option to email asset owners (experimental). <Image
```

Comment:
Adds the email asset owners notification policy to the asset-based alert section